jangjuseong/services/userService.js

```diff
@@ -0,0 +1,37 @@
+const userDao = require('../models/userDao');
+
+const bcrypt = require('bcrypt');
+const jwt = require('jsonwebtoken');
+
+const SECRET_KEY = process.env.SECRET_KEY;
+
+const signUp = async (name, email, password, profileImage) => {
+  const saltRounds = 12;
+  const hashPassword = await bcrypt.hash(password, saltRounds);
+  const createUser = await userDao.createUser(
+    name,
+    email,
+    hashPassword,
+    profileImage
+  );
+
+  return createUser;
+};
+
+const signIn = async (email, password) => {
+  const [userData] = await userDao.signInUser(email);
+
+  const payLoad = { userId: userData.id };
+
+  const checkHash = await bcrypt.compare(password, userData.password);
+  if (!checkHash) {
+    return 'invalidUser';
+  }
+
+  return jwt.sign(payLoad, SECRET_KEY);
```

Comment:
바로 return하는 방법도 좋고 아래 방법도 좋으니 참고해주세요!
```suggestion
  const token = jwt.sign(payload, SECRET_KEY);
  return token
```